build.sbt

```diff
@@ -33,6 +33,7 @@ lazy val plugin = project
     name := "sbt-play-ebean",
     organization := "com.typesafe.sbt",
     libraryDependencies ++= sbtPlayEbeanDeps,
+    crossScalaVersions := Seq(),
```

Comment:
@jroper I'm afraid this is plain wrong, but I can't find a better way of fixing it.